lunar/distribution.yaml

```diff
@@ -238,11 +238,20 @@ repositories:
       version: kinetic-devel
     status: maintained
   catkin_pip:
+    doc:
+      type: git
+      url: https://github.com/pyros-dev/catkin_pip.git
+      version: devel
     release:
       tags:
         release: release/lunar/{package}/{version}
       url: https://github.com/pyros-dev/catkin_pip-release.git
-      version: 0.2.2-0
+      version: 0.2.3-0
+    source:
+      test_pull_requests: true
```

Comment:
I just want to confirm that you have provided the necessary access for the @ros-pull-request-builder ?
([prompted by bloom](https://github.com/ros-infrastructure/bloom/blob/d48aa0277639952e250706b54f1d3bbc786c395e/bloom/commands/release.py#L533-L536) when you selected "test_pull_requests")